pages/c/_cluster/apps/charts/install.vue

```diff
@@ -103,6 +104,12 @@ export default {
       this.forceNamespace = null;
     }
 
+    const isMultiClusterApp = await this.existing.deployedAsMultiCluster();
```

Comment:
You should be able to just do `this.mcapp = await this.existing.deployedAsMultiCluster();`